.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: Create Release
+on:
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3.0.2
+      - name: Unit tests
+        uses: ./.github/actions/unit-tests
+
+  release:
+    needs: test
+    name: Release
+    uses: keptn/gh-automation/.github/workflows/release-integration.yml@v1.5.3
+
+  docker_build:
+    needs: [release]
+    name: Docker Build
+    runs-on: ubuntu-20.04
+    env:
+      VERSION: ${{ needs.release.outputs.RELEASE_TAG }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3.0.2
+
+      - name: Load CI Environment from .ci_env
+        id: load_ci_env
+        uses: c-py/action-dotenv-to-setenv@v3
+        with:
+          env-file: .ci_env
+
+      - name: Sanitize repo owner
+        uses: actions/github-script@v4
+        id: repo_slug
+        with:
+          result-encoding: string
+          script: return '${{ github.repository_owner }}'.toLowerCase()
+
+      - name: Docker Build
+        uses: keptn/gh-automation/.github/actions/docker-build@v1.5.3
+        with:
+          TAGS: |
+            ghcr.io/${{ steps.repo_slug.outputs.result }}/${{ env.IMAGE }}:${{ env.VERSION }}
+          BUILD_ARGS: |
+            version=${{ env.VERSION }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  helm_chart_build:
+    needs: [release, docker_build]
+
+    name: Build Helm Charts
+    runs-on: ubuntu-20.04
+    env:
+      VERSION: ${{ needs.release.outputs.RELEASE_TAG }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3.0.2
+
+      - name: Load CI Environment from .ci_env
+        id: load_ci_env
+        uses: c-py/action-dotenv-to-setenv@v3
+        with:
+          env-file: .ci_env
+
+      - name: Build Helm Charts
+        id: build_helm_charts
+        uses: keptn/gh-automation/.github/actions/build-helm-charts@v1.6.0
+        with:
+          VERSION: ${{ env.VERSION }}
+          APP_VERSION: ${{ env.VERSION }}
+          CHART_NAME: ${{ env.IMAGE }}
+
+      - name: Upload Helm Chart as release asset
+        env:
+          RELEASE_TAG: ${{ needs.release.outputs.RELEASE_TAG }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release upload "$RELEASE_TAG" installer/*.tgz
```

Comment:
In Keptn we have `--draft` to have a final human check that verifies everything was correct before publishing the release. Do you think would it be worth it here as well or not?

Reply:
There is a draft release aswell, this line is just for uploading the release asset